src/h3/api/_api_template.py

```diff
@@ -754,4 +754,20 @@ def get_pentagon_indexes(resolution):
 
         return _out_unordered(mv)
 
+    def get_res0_indexes():
+        """
+        Return all cells at a given resolution.
```

Comment:
This only returns the base cells, right?

Reply:
good catch!